parser/parsetypes/http.go

```diff
@@ -10,73 +10,79 @@ type HTTP struct {
 	// response pair
-	Tags []string `bson:"tags" bro:"tags" brotype:"set[enum]"`
+	Tags []string `bson:"tags" bro:"tags" brotype:"set[enum]" json:"tags"`
 	// UserName will contain a username in the case of basic auth implementation
-	UserName string `bson:"username" bro:"username" brotype:"string"`
+	UserName string `bson:"username" bro:"username" brotype:"string" json:"username"`
 	// Password will contain a password in the case of basic auth implementation
-	Password string `bson:"password" bro:"password" brotype:"string"`
+	Password string `bson:"password" bro:"password" brotype:"string" json:"password"`
 	// Proxied contains all headers that indicate a request was proxied
-	Proxied []string `bson:"proxied" bro:"proxied" brotype:"set[string]"`
+	Proxied []string `bson:"proxied" bro:"proxied" brotype:"set[string]" json:"proxied"`
 	// OrigFuids contains an ordered vector of uniq file IDs
-	OrigFuids []string `bson:"orig_fuids" bro:"orig_fuids" brotype:"vector[string]"`
+	OrigFuids []string `bson:"orig_fuids" bro:"orig_fuids" brotype:"vector[string]" json:"orig_fuids"`
 	// OrigFilenames contains an ordered vector of filenames from the client
-	OrigFilenames []string `bson:"orig_filenames" bro:"orig_filenames" brotype:"vector[string]"`
+	OrigFilenames []string `bson:"orig_filenames" bro:"orig_filenames" brotype:"vector[string]" json:"orig_filenames"`
 	// OrigMimeTypes contains an ordered vector of mimetypes
-	OrigMimeTypes []string `bson:"orig_mime_types" bro:"orig_mime_types" brotype:"vector[string]"`
+	OrigMimeTypes []string `bson:"orig_mime_types" bro:"orig_mime_types" brotype:"vector[string]" json:"orig_mime_types"`
 	// RespFuids contains an ordered vector of unique file IDs in the response
-	RespFuids []string `bson:"resp_fuids" bro:"resp_fuids" brotype:"vector[string]"`
+	RespFuids []string `bson:"resp_fuids" bro:"resp_fuids" brotype:"vector[string]" json:"resp_fuids"`
 	// RespFilenames contains an ordered vector of unique files in the response
-	RespFilenames []string `bson:"resp_filenames" bro:"resp_filenames" brotype:"vector[string]"`
+	RespFilenames []string `bson:"resp_filenames" bro:"resp_filenames" brotype:"vector[string]" json:"resp_filenames"`
 	// RespMimeTypes contains an ordered vector of unique MIME entities in the HTTP response body
-	RespMimeTypes []string `bson:"resp_mime_types" bro:"resp_mime_types" brotype:"vector[string]"`
+	RespMimeTypes []string `bson:"resp_mime_types" bro:"resp_mime_types" brotype:"vector[string]" json:"resp_mime_types"`
 }
 
 //TargetCollection returns the mongo collection this entry should be inserted
-//into
-func (line *HTTP) TargetCollection(config *config.StructureTableCfg) string {
+func (in *HTTP) TargetCollection(config *config.StructureTableCfg) string {
 	return config.HTTPTable
 }
 
 //Indices gives MongoDB indices that should be used with the collection
-func (line *HTTP) Indices() []string {
+func (in *HTTP) Indices() []string {
```

Comment:
Did these get changed to "in" to match with the others parsetypes? What if we just change all of them to "line", it seems to be clearer to me...

Reply:
Yep, and can do!